app/presenters/deposit_title_presenter.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# A helper for displaying a title for a Work or Collection
+class DepositTitlePresenter
+  def self.show(deposit)
+    if deposit.instance_of?(WorkVersion)
+      deposit&.title.presence || Settings.h2.no_title
+    elsif deposit.instance_of?(CollectionVersion)
+      deposit&.name.presence || Settings.h2.no_title
+    end
+  end
+end
```

Comment:
refactored and renamed this from `WorkTitlePresenter` so it can be used for works or collections

Reply:
I'm not sure there is commonalty enough to introduce  this abstraction. This adds an additional conditional every time it's called, even though we typically already know what type it is.

Reply:
Revert the `WorkTitlePresenter` back to it's current form, added a new `CollectionTitlePresenter` to use in other parts of the code too, and then added this conditional to the `ContinueDepositModalComponent` to pick the right one.